scripts/jmehats_JEC.py

```diff
@@ -100,6 +100,10 @@
 	print bcolors.FAIL+"\tNo need to use the patJetUpdater."+bcolors.ENDC
 	exit(-7)
 
+dataTier = "AOD"
```

Comment:
What was wrong with the way it was done before? Just trying to understand the impact of this change.

Reply:
For the record: The toolbox can now run on AOD/miniAOD/nanoAOD, so the jettoolbox-config changed from a bool to a dataTier-string 👍